tests/spec/fs.write.spec.js

```diff
@@ -65,4 +65,49 @@ describe('fs.write', function() {
       });
     });
   });
+
+  it('should fail to write to a file that does not exist', function(done) {
+    const fs = util.fs();
+    const buffer = Buffer.from([1, 2, 3, 4, 5, 6, 7, 8]);
+
+    fs.write('/myfile', buffer, 0, buffer.length, 0, function(error) {
```

Comment:
`fs.write` takes an `fd`, so passing it a filepath as a `string` is really testing what we do when we don't pass an `fd`.  If I do this in node, it throws.  Probably returning `EBADF` is OK here, since it literally is a bad file descriptor.

That said, let's adjust the message on line 69 to be more clear: `it('should fail if given a file path vs. an fd'`